additional_explorations.md

```diff
@@ -215,14 +499,330 @@ const palette = window.open("/palette", "palette", "dependent=true");
+
+```js
+self.addEventListener('screenschange', function(event) {
+  // Ensure the best screen is used for fullscreen on screen change events.
+  // NEW: screenschange event handlers may call `requestFullscreen()`.
+  if (document.fullscreenElement)
+    document.body.requestFullscreen({ screen: getBestScreen() });
+});
+```
+
+Since user-generated screen change events are not user activations of the site,
+we suggest specifically allowing Element.requestFullscreen() when the algorithm
+is triggered by a user-generated screen change. This parallels the existing
+[spec](https://fullscreen.spec.whatwg.org/#dom-element-requestfullscreen)
+allowance when triggered by a "user generated orientation change".
+
+### Working with inner size versus outer size
+
+window.open() treats supplied width and height feature string values as the
+inner/content size, while window.resizeTo() treats supplied width and height
+values as outer/window sizes. Ideally developers could supply either inner or
+outer dimensions for either method.
+
+There is likely no meaningful reason for a site to supply the inner location of
+a window, (i.e. the top and left screen coordinates of the content, inset inside
+the window frame).
+
+### Window placement limitations
+
+Changes allowing cross-screen window placements should still be subject to
+reasonable limitations, and some may be left to the discretion of implementers.
+
+Implementers generally restrict window movement to certain windows types, (e.g.
+standalone web app windows and popup windows, not tabbed browser windows) and
+only under certain circumstances (e.g. secure contexts, with transient user
+activation, maybe limiting the number or frequency of interactions, etc.).
+
+Window placements could be clamped to the available bounds of a single target
+screen, or perhaps allowed to span multiple screens. It seems reasonable for
+implementers to restrict window placements that would make some portion of
+the window appear outsides the aggregate available screen bounds.
+
+It may be valuable to provide clearly specified behaviors for some cases, or to
+offer non-normative guidance for how such cases could be handled.
+
+### Respecting intents around user-agent-defined behaviors
+
+Reasonable caution should be exercised when changing implementation-specific
+behavior of existing Web Platform APIs, to avoid breaking existing users. The
+leading proposal aims to align behavior around the existing behavior of some
+implementers, and honor perceived intent.
+
+Still, it may be prudent for implementers to retain elements of old behavior
+that sites might rely upon. For example, if developers expect window placements
+to be clamped within the current `Screen` and supply bounds outside the overall
+`Screen` space (e.g. "left=99999999"), then it may be reasonable to clamp
+window placements within the current `Screen`, rather than within the `Screen`
+nearest those coordinates.
+
+### Changes to `colorDepth` and `pixelDepth`
```

Comment:
This looks very similar to the section starting on line 664

Reply:
Removed, thanks!